--- a/mxm_wifiex/wlan_src/Makefile
+++ b/mxm_wifiex/wlan_src/Makefile
@@ -364,6 +364,7 @@ endif
 #ccflags-y += -Wmisleading-indentation
 #ccflags-y += -Wunused-const-variable
 ccflags-y += -Wno-array-bounds
+ccflags-y += -Wno-unused-variable
 #############################################################################
 # Make Targets
 #############################################################################
